includes/event/event-repository.php

```diff
@@ -501,10 +504,14 @@ private function get_event_post( int $event_id ): ?WP_Post {
 	/**
 	 * @throws Exception
 	 */
-	private function get_event_meta( int $event_id ): array {
+	private function get_event_meta( int $event_id ): ?array {
 		$meta = get_post_meta( $event_id );
 		$utc  = new DateTimeZone( 'UTC' );
 
+		if ( ! isset( $meta['_event_start'][0], $meta['_event_end'][0], $meta['_event_timezone'][0] ) ) {
```

Comment:
Why the `[0]`?

Reply:
When we have data, this is an example of the `$meta` content:

```php
array(
  "_event_start" => array(
    0 => "2024-05-01 00:02:00"
  ),
  "_event_end" => array(
    0 => "2024-05-31 23:58:00"
  ),
  "_event_timezone" => array(
    0 => "Europe/Luxembourg"
  ),
  "_hosts" =>array(
    0 => ""
  ),
)
```
so we need to access to this `[0]` element.

Reply:
Ok understood, thank you! Since we're not specifcying the third parameter $single of get_post_meta it will always return an array.